src/plugins/visualizations/public/embeddable/_embeddables.scss

```diff
@@ -45,4 +45,21 @@
   .visLegend__toggle {
     opacity: 1; /* 3 */
   }
+}
+
+.embPanel {
+  .visualize[data-loading],
+  .visualize[data-error] {
+    pointer-events: none;
+    -webkit-filter: grayscale(100%);
+    -moz-filter: grayscale(100%);
+    -ms-filter: grayscale(100%);
+    -o-filter: grayscale(100%);
```

Comment:
wonder if vendor prefixes are needed or if this is handed automatically on build step

Reply:
@elastic/eui-design ?

Reply:
@Dosant I see that in some places using more advanced css features, we do use the vendor prefixes. For example in `x-pack/plugins/ml/public/application/components/job_selector/_job_selector.scss`

Reply:
Our SASS compiler **does** use an auto-prefixer so you only need the generic `filter` attribute. This way it also stays current with the browsers we support.